micro-application-register/build.gradle

```diff
@@ -16,7 +16,7 @@ modifyPom {
 		url 'https://github.com/aol/micro-server'
```

Comment:
Should we change the github repo as well, or leave it in /aol/micro-server ?

Reply:
There is no Oath OSS github repo yet.